src/types/field/emailField.ts

```diff
@@ -15,4 +15,6 @@ export interface IEmailField extends IField {
 
 export interface IEmailFieldSchema extends IEmailField, IFieldSchema {
   isVerifiable: boolean
+  hasAllowedEmailDomains: boolean
```

Comment:
These keys should be in the `IEmailField` schema, not sure why `isVerifiable` is in this interface too, can remove it for me? Thank you!

Reply:
So I move `hasAllowedEmailDomains` and the rest into `IEmailField` too? Gotcha

Reply:
I can't remove `isVerifiable` from `IEmailFieldSchema`.

Because there is a `isVerifiable` in `IEmailField` and `isVerifiable?` in `IFieldSchema`.

The resulting error was `error TS2320: Interface 'IEmailFieldSchema' cannot simultaneously extend types 'IEmailField' and 'IFieldSchema'.`

Reply:
Yea, that was probably why it was in there in the first place then haha! Thanks for moving the others